packages/manager/apps/pci-quota/src/pages/quota/IncreaseQuota.page.tsx

```diff
@@ -29,173 +21,81 @@ export default function IncreaseQuotaPage(): JSX.Element {
-            }
+            {(_t) => (
+              <span
+                dangerouslySetInnerHTML={{
+                  __html: _t(
+                    'pci_projects_project_quota_increase_buy_success_message',
+                    {
+                      billingUrl: url,
+                    },
+                  ),
+                }}
+              ></span>
+            )}
           </Translation>,
         );
-      } else {
-        setIsLoading(true);
-        trackClick({
-          name: `${TRACK.BASE}::${TRACK.CONTACT_SUPPORT}::${TRACK.CONFIRM}`,
-          type: 'action',
-        });
-        try {
-          const { ticketId } = await createTicket({
-            issueTypeId: issueType.id,
-            serviceName: projectId,
-            subject: issueType.label,
-            body: `
-${issueType.subject}
-
-${inputLabel}
-
-${formData}
-      `,
-          });
-          trackPage({
-            name: `${TRACK.BASE_CONTACT_SUPPORT_BANNER}::${TRACK.SUCCESS}`,
-          });
-          addSuccess(
-            <Translation ns="quotas/increase">
-              {(_t) => (
-                <span
-                  dangerouslySetInnerHTML={{
-                    __html: _t(
-                      'pci_projects_project_quota_increase_success_message',
-                      {
-                        ticketUrl:
-                          SUPPORT_TICKET_ID_URL.replace(
-                            '{ticketId}',
-                            ticketId,
-                          ) + me.ovhSubsidiary,
-                      },
-                    ),
-                  }}
-                ></span>
-              )}
-            </Translation>,
-          );
-          goBack();
-        } catch (e) {
-          trackPage({
-            name: `${TRACK.BASE_CONTACT_SUPPORT_BANNER}::${TRACK.ERROR}`,
-          });
-          addError(
-            <Translation ns="quotas/increase">
-              {(_t) =>
-                _t('pci_projects_project_quota_increase_error_message', {
-                  message: (e as { data: { message: string } }).data?.message,
-                })
-              }
-            </Translation>,
-          );
-          goBack();
-        } finally {
-          setIsLoading(false);
-        }
-      }
-    } else if (type === Type.CREDIT) {
-      const planCode = formData || 'quota-no-plan';
-
-      const serviceOption = serviceOptions.find((s) => s.planCode === planCode);
-
-      if (!serviceOptions) {
+        goBack();
+      } catch (e) {
         trackPage({
-          name: `${TRACK.BASE_SELECT_PLAN_BANNER}::${TRACK.ERROR}::${planCode}`,
+          name: `${TRACK.BASE_SELECT_PLAN_BANNER}::${TRACK.SUCCESS}::${planCode}`,
         });
         addError(
           <Translation ns="quotas/increase">
             {(_t) =>
               _t('pci_projects_project_quota_increase_error_message', {
-                message: '',
+                message: (e as { data: { message: string } }).data?.message,
```

Comment:
Pretty weak to type cast here, I'll suggest to use a type guard to ensure that `error is ApiError`